packages/salesforcedx-vscode-core/src/orgBrowser/browser.ts

```diff
@@ -44,7 +44,8 @@ export class OrgBrowser {
     const username = await OrgAuthInfo.getDefaultUsernameOrAlias(false);
     this._dataProvider = new MetadataOutlineProvider(username);
     this._treeView = window.createTreeView(OrgBrowser.VIEW_ID, {
-      treeDataProvider: this._dataProvider
+      treeDataProvider: this._dataProvider,
+      canSelectMany: true
```

Comment:
still coming up to speed. Is there any we're doing with the org view where making this multiselect will be a problem?

Reply:
@jag-sfdc do you have any concerns that you can think of here? I'm not seeing any errors with a spot check, but some interesting behavior with Diff and Delete (only takes the selected directory) and Retrieve and Deploy work well together.

Reply:
Diff and Deploy/Retrieve would be the areas I would be worried about. If you have tested those, we should be good I hope.